system/jlib/jarray.hpp

```diff
@@ -27,7 +27,7 @@
 
 typedef size32_t aindex_t;
 
-#define NotFound   (aindex_t)-1
```

Comment:
Was clashing with something in 3rd-patry cuckoo filter code I was trying out. Decided not to use it but this change is still a good one I think.